contrib/automation_tests/README.md

```diff
@@ -1,7 +1,21 @@
-This directory contains end-to-end tests for Orbit. 
+This directory contains end-to-end tests for Orbit.
+
+# Installing the prerequesits
 
 We use pywinauto to remote control the Orbit UI. For each test there is a script $TestName.py in this folder.  
-You need a python environment to run the automation scripts. Install the dependencies:
+You need a python environment to run the automation scripts. 
+
+Note that the use of virtual environments is recommended: https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/
+
+To setup a virtual environment in Python3, run the following inside this folder:
+
+```
+python -m venv env
```

Comment:
Nit: I'd use 'orbit-e2e-tests' instead of 'env'

Reply:
This is how it's suggested in the documentation and I wanted to match that since I linked to it above. This is created as a local folder, so it's quite common to just name it "env" or "venv" AFAIK

Reply:
Oh, these are local to the directories! Wasn't aware of that.